2 dz/3/main.cpp

```diff
@@ -0,0 +1,37 @@
+/*Напечатать все представления натурального числа N суммой натуральных слагаемых. Перестановка слагаемых нового способа не дает.*/
+
+#include <iostream>
+
+using namespace std;
+
+void the_executor(int *a, int n1, int pos, int N)
```

Comment:
Style guide p. 11

Reply:
Probably pos is a position, but it isn't so obvious about n1 and N. Maybe there can be more readable argument names?